brains/andalite.py

```diff
@@ -79,18 +90,21 @@ def threadMain(self):
 
     async def messageConsumer(self, websocket):
         while True:
-            message = await websocket.recv()
-            data = ast.literal_eval(message)
-            print("received data:")
-            pprint.pprint(data)
-            self.inboundMemoryWrites.put_nowait(data)
+            messageString = await websocket.recv()
+            message = ast.literal_eval(messageString)
+            num = message.get('event_number', -1)
+            if num > self.currentEventNumber:
+                self.currentEventNumber = num
+                visualization.textlog.log_text(f"Accepted event with num {num}")
+                self.inboundMemoryWrites.put_nowait(message.get('data'))
+            else:
+                visualization.textlog.log_text(f"Rejected event with num {num} < {self.currentEventNumber}")
 
     async def messageProducer(self, websocket):
         while True:
             try:
                 message = await asyncio.get_event_loop().run_in_executor(None, self.bgThreadGetOutboundMemoryReads)
-                print("sending a message: {}".format(message))
-                pprint.pprint(message)
+                visualization.textlog.log_text(f"Sending event with num {self.currentEventNumber}")
```

Comment:
once we have 'smart' strategies we should describe what the event actually was here too